xamarin/Xamarin.Forms
# [Android] Fix NRE changing SwipeItem visibility

### Description of Change ###

Fix NRE changing SwipeItem visibility.

### Issues Resolved ### 

- fixes #10530 

### API Changes ###
 
 None

### Platforms Affected ### 
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the Issue10530. Press the SwipeItem to change the visibility and verify that everything works as expected.

### PR Checklist ###
